package.json

```diff
@@ -70,7 +70,9 @@
     "@types/lodash": "^4.14.166",
     "@types/node": "^14.14.14",
     "@types/react": "^18.3.1",
+    "@types/react-color": "^3.0.12",
     "@types/react-dom": "^18.3.0",
+    "@types/styled-components": "^5.1.34",
```

Comment:
Has the incredibly annoying bug that slows VS code 100000% been fixed yet? That's the reason I removed the styled-components types

Reply:
I think it got fixed since i didn't experience any slowness